src/sentry/tasks/store.py

```diff
@@ -175,11 +174,10 @@ def _do_preprocess_event(
         ):
             reprocessing2.backup_unprocessed_event(data=original_data)
 
-            is_low_priority = should_demote_symbolication(first_platform, project_id)
             submit_symbolicate(
                 SymbolicatorTaskKind(
                     platform=first_platform,
-                    is_low_priority=is_low_priority,
+                    is_low_priority=False,
```

Comment:
Can we remove this field altogether or it would cause problems?

Reply:
It's [defaulted](https://github.com/getsentry/sentry/blob/f78327b265f19f18f562d78400eeb0a7da708c0f/src/sentry/lang/native/symbolicator.py#L50), so we can remove it here. I would delete it in a separate PR.